app/main.go

```diff
@@ -271,6 +274,13 @@ func main() {
 		}
 	}
 
+	osApiKey, err := secretsClient.OrdnanceSurveyApiKey()
+	if err != nil {
+		logger.Fatal(err)
+	}
+
+	addressClient := ordnance_survey.NewClient(ordnanceSurveyBaseUrl, osApiKey, http.DefaultClient)
```

Comment:
it would make more sense to have this return `*Client` instead of taking the address below

Reply:
Ah yeah makes sense - still getting used to pointers and references.